Makefile

```diff
@@ -55,7 +55,7 @@ conform:
 lint:        ## Lint code.
              ##
 	@echo ">> linting code"
-	DOCKER_BUILDKIT=1 docker run --rm -v $(shell pwd):/app -w /app golangci/golangci-lint:v1.54.2 golangci-lint run -v --timeout=5m
+	go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.2 run --verbose --timeout 5m
```

Comment:
suggestion: we've been using `NO_DOCKER` in the Makefile for fast-paths. To keep current conventions, consider using that type of branch here.